### ISSUE TITLE
Publish new version with framework path fix

### ISSUE THREAD

**Issue by emoon** [RustAudio/coreaudio-sys#10]
Hi,

I wonder if it would be possible to publish a new version that includes the framework path fix https://github.com/RustAudio/coreaudio-sys/pull/6 ? Currently https://github.com/tomaka/rodio latest version doesn't compile because of the issue (Rodio uses cpal which uses coreaudio-rs that uses coreaudio-sys)

**Comment by torkleyy** [comment]
@yupferris It seems you have access to https://crates.io/crates/coreaudio-sys, too. If you could do a release of this crate, that would be great. We're all blocked by this, I don't want to release my crate when users need to add patch sections to their `Cargo.toml`. Thanks in advance!

**Comment by mitchmindtree** [comment]
@Rhuagh thanks a lot for your patience and following up with these issues in my absense! I returned from the Himalayas about 24hrs ago but probably won't get a chance to start working through the backlog of issues til tomorrow. I've added you as a collaborator to this repository and have added you as a publisher for the crate on crates.io. Thanks a lot for reaching out and taking the time to help!
